Documentation.md

```diff
@@ -370,11 +373,11 @@ be called on behalf of the agent. There is only ever one agent per Streams
 instantiation and all contacts and actions are assumed to be taken on behalf of
 this one agent.
 
-### `agent.onContactPending()`
+### `agent.onContactPending()` -- DEPRECATED
```

Comment:
Can you add `@deprecated` to the [method](https://github.com/amazon-connect/amazon-connect-streams/blob/60b2e86d497f51caaa7a2306e8a9320b57c772e9/src/api.js#L275)

Reply:
done